blueprint/index.md

```diff
@@ -0,0 +1,254 @@
+
+The Event Orchestration trigger invokes these workflows. The workflows in turn calls the Disconnect Voice Call and Put Conversation Tag data actions to update the outbound phone call.
+
+First import this workflow to your Genesys Cloud organization:
+
+1. Download the `Terminate Outbound Call Missing Queue.i3WorkFlow` file from the [terminate-voice-calls-with-no-queue repo](https://github.com/GenesysCloudBlueprints/terminate-voice-calls-with-no-queue) GitHub repository.
+
+:::primary
+  **Note:** If you would like to still allow Communicate/PBX calls between GC Users, use the `Terminate Outbound Call Missing Queue with PSTN Call Leg Check.i3WorkFlow` file.
+  :::
+
+2. In Genesys Cloud, navigate to **Admin** > **Architect** > **Flows:Workflow** and click **Add**.
+
+   ![Import the workflow](images/AddWorkflow1.png "Import the workflow")
+
+3. Enter a name for the workflow and click **Create Flow**.
+
+   ![Name your workflow](images/NameWorkflow1.png "Name your workflow")
+
+4. From the **Save** menu, click **Import**.
+
+   ![Import the workflow](images/ImportWorkflow1.png "Import the workflow")
+
+5. Select the downloaded **Terminate Outbound Call Missing Queue.i3WorkFlow** file and click **Import**.
+
+   ![Import your workflow file](images/SelectWorkflow1ImportFile.png "Import your workflow file")
+
+6. Review your workflow. Click **Save** and then click **Publish**.
+![Save your workflow](images/ImportedWorkflow1.png "Save your workflow")
+:::primary Note: If you would like to change the External Tag, replace **No Queue** in the **externalTagName** field with the string of your choice. :::
+
+:::primary Note: If you imported the `Terminate Outbound Call Missing Queue with PSTN Call Leg Check.i3WorkFlow` file, your workflow will look like the screenshot below. :::
+
+![Save your workflow](images/ImportedWorkflow2.png "Save your workflow")
+
+:::primary Note: If you imported the `Terminate Outbound Call Missing Queue with PSTN Call Leg Check.i3WorkFlow` file, your workflow will look like the screenshot above. :::
+
+## Create the event orchestration triggers
+
+Create the trigger that invokes the created Architect workflow.
+
+1. From Admin Home, search for **Triggers** and navigate to the Triggers list.
+
+   ![Navigate to Triggers](images/NavigateToTriggers.png "Navigate to Triggers")
+
+2. From the Triggers list, click **Add Trigger**
+
+   ![Add Trigger](images/AddTrigger.png "Add Trigger")
+
+3. From the Add New Trigger modal, name your trigger and click **Add**
+
+   ![Name Trigger](images/NameTrigger.png "Name Trigger")
+
+4. From the Trigger single view, in the **Topic Name** menu, select **v2.detail.events.conversation.{id}.user.start**.  In the **Workflow Target** menu, select **Terminate Outbound Call Missing Queue**.  Leave **Data Format** as **TopLevelPrimitives**.  Click **Add Condition**.
```

Comment:
Jason you might want to reference or link to the notifications topics page here.  https://developer.genesys.cloud/notificationsalerts/notifications/available-topics

Also it might be good to have a call out that you can actually watch the notifications for the user happen by using our notification monitoring tool in the Developer Center.

Reply:
@carnellj-genesys I have made this change